public/form_validation.js

```diff
@@ -0,0 +1,43 @@
+let elements = document.querySelectorAll('p.required');
+let inputs = document.querySelectorAll('p.required>input');
+let selects = document.querySelectorAll('p.required>select');
+            
+for(let i=0; i<elements.length; i++){
+        let el = elements[i];
+        let input = el.querySelector('input');
+        if(input){
+        input.onblur = function() {        
```

Comment:
@Varannchik  if the validation function for the input is the same as for the select is it better to create a function expression?

Reply:
Updated